components/signuppage.tsx

```diff
@@ -0,0 +1,164 @@
+"use client";
+
+import React, { useState, ChangeEvent } from 'react';
+import { Passwordreveal } from '@/assets/icons';
+import { Googleicon } from '@/assets/icons';
+import { Facebookicon } from '@/assets/icons';
+import { Vectorline } from '@/assets/icons';
+
+const Signupform = () => {
```

Comment:
Use `SignupForm` this name instead of `Signupform`. You should also rename this file to `signup-form`